tests/vm_test.go

```diff
@@ -1829,8 +1830,16 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				By("Waiting for VMI to start")
 				waitForVMIStart(virtClient, vmi)
 
+				By("Getting current vmi instance")
+				originalVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &k8smetav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				var args = []string{vm.COMMAND_STOP, "--namespace", thisVm.Namespace, thisVm.Name, "--dry-run"}
+				if flags != nil {
+					args = append(args, flags...)
+				}
```

Comment:
nit: I think this could be simplified like so

```suggestion
				args := append([]string{vm.COMMAND_STOP, "--namespace", thisVm.Namespace, thisVm.Name, "--dry-run"}, flags...)
```

Reply:
You are right, I was worried about the resize of the slice, that obviously will not gonna happens if `flags` is `nil`. Thanks so much